tests/mock/vscode.ts

```diff
@@ -193,6 +193,8 @@ export class TestItem {
   tags: readonly TestTag[] = [];
   canResolveChildren = false;
   status: 'none' | 'enqueued' | 'started' | 'skipped' | 'failed' | 'passed' = 'none';
+  _didChangeStatus = new EventEmitter<TestItem>();
+  onDidChangeStatus = this._didChangeStatus.event;
```

Comment:
I don't see this on vscode.TestItem,

Reply:
I'll use `TestController.onDidCreateTestRun > TestRun.onDidChange` to achieve the same goal.